env/ipp_aes_ctr_provider.h

```diff
@@ -0,0 +1,89 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  Copyright (c) 2020 Intel Corporation
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#if !defined(ROCKSDB_LITE)
+
+#ifdef IPPCP
+
```

Comment:
Can you add a comment in here about the IPPCP provider in general?  What it does, where to get it, etc.  A header comment for the file and/or class would be helpful.